crates/red_knot_python_semantic/src/semantic_index/definition.rs

```diff
@@ -210,13 +211,9 @@ pub(crate) struct MatchPatternDefinitionNodeRef<'a> {
     pub(crate) index: u32,
 }
 
-impl DefinitionNodeRef<'_> {
+impl<'db> DefinitionNodeRef<'db> {
```

Comment:
Nit: I would use `'a` here to be consistent with the `DefinitionNodeRef` definition and we don't use db here

Reply:
I think the main reason to use `'db` here is because I've used that in `DefinitionKind` as well which is the return type of `DefinitionNodeRef::into_owned` function.